Octokit/Models/Response/OauthToken.cs

```diff
@@ -20,6 +20,19 @@ public OauthToken(string tokenType, string accessToken, IReadOnlyList<string> sc
             this.ErrorUri = errorUri;
         }
 
+        public OauthToken(string tokenType, string accessToken, int expiresIn, string refreshToken, int refreshTokenExpiresIn, IReadOnlyList<string> scope, string error, string errorDescription, string errorUri)
```

Comment:
Do you think it'd be worthwhile to add a comment here and above describing the situations when you'd want to use each constructor?

Reply:
I primarily kept the old one, to keep the change binary compatible - to avoid breaking changes :)
Added a summary for good measure.

Reply:
Got it, thank you! I think it's reasonable to mark the old one as deprecated to be removed in a future major release. After that's done, I can merge and release this!